src/DegenBox.sol

```diff
@@ -55,6 +56,8 @@ contract MasterContractManager is BoringOwnable, BoringFactory {
     // solhint-disable-next-line var-name-mixedcase
     uint256 private immutable DOMAIN_SEPARATOR_CHAIN_ID;
 
+    mapping(IERC20 => bool) allowedBrokenTotalSupplyTokens;
```

Comment:
what is this exactly?

Reply:
is this necessary now that we are working with the working staked GLP?

Reply:
I forgot to remove it